app/models/parse_utils.rb

```diff
@@ -410,7 +410,7 @@ def self.initialize_coordinate_label_data_arrays(study, coordinate_file, user, o
       end
 
       raw_header_data = coordinate_data.readline.encode('UTF-8', invalid: :replace, undef: :replace, replace: '').split(/[\t,]/).map(&:strip)
-      header_data = study.sanitize_input_array(raw_header_data)
+      header_data = self.sanitize_input_array(raw_header_data)
```

Comment:
Ahhh yes...  this particular issue has raised its head many times due to various refactors.  Good catch.